content/en/docs/concepts/security/multi-tenancy.md

```diff
@@ -121,7 +121,7 @@ Data plane isolation ensures that pods and workloads for different tenants are s
 
 By default, all pods in a Kubernetes cluster are allowed to communicate with each other, and all network traffic is unencrypted. This can lead to security vulnerabilities where traffic is accidentally or maliciously sent to an unintended destination, or is intercepted by a workload on a compromised node.
 
-Pod-to-pod communication can be controlled  using [Network Policies](/docs/concepts/services-networking/network-policies/), which restrict communication between pods using namespace labels or IP address ranges. In a multi-tenant environment where strict network isolation between tenants is required,   starting with a default policy that denies communication between pods is recommended with another rule that allows all pods to query the DNS server for name resolution. With such a default policy in place, you can begin adding more permissive rules that allow for communication within a namespace. This scheme can be further refined as required. Note that this only applies to pods within a single control plane; pods that belong to different virtual control planes cannot talk to each other via Kubernetes networking.
+Pod-to-pod communication can be controlled using [Network Policies](/docs/concepts/services-networking/network-policies/), which restrict communication between pods using namespace labels or IP address ranges. In a multi-tenant environment where strict network isolation between tenants is required, starting with a default policy that denies communication between pods is recommended with another rule that allows all pods to query the DNS server for name resolution. With such a default policy in place, you can begin adding more permissive rules that allow for communication within a namespace. It is also recommended not to use empty label selector '{}' for namespaceSelector field in network policy definition in case traffic need to be allowed between namespaces.This scheme can be further refined as required. Note that this only applies to pods within a single control plane; pods that belong to different virtual control planes cannot talk to each other via Kubernetes networking.

```

Comment:
How I think I'd write this:
```suggestion
Pod-to-pod communication can be controlled using [Network Policies](/docs/concepts/services-networking/network-policies/), which restrict communication between pods using namespace labels or IP address ranges. In a multi-tenant environment where strict network isolation between tenants is required, starting with a default policy that denies communication between pods is recommended with another rule that allows all pods to query the DNS server for name resolution. With such a default policy in place, you can begin adding more permissive rules that allow for communication within a namespace. For [namespace](#namespaces) isolation between tenants, consider whether you need to automatically apply NetworkPolicy restrictions and block cross-namespace traffic. If you use namespaces to isolate tenants, you may also need to avoid or block an empty namespace selector ('{}') within a NetworkPolicy.  This scheme can be further refined as required. Note that this only applies to pods within a single control plane; pods that belong to different virtual control planes cannot talk to each other via Kubernetes networking (but may be able to communicate over the internet, or other networking outside Kubernetes' control).
```

Reply:
@sftim  Thanks for useful suggestion. As all the traffic between namespaces are by default denied. The recommendation added is for the use case where traffic between namespaces need to be allowed.  What i think would be more appropriate is:

For [namespace](#namespaces) isolation between tenants, consider whether you need to automatically apply NetworkPolicy restrictions and block cross-namespace traffic. If traffic needs to be allowed between namespaces owned by a tenant, you may also need to avoid or block an empty namespace selector ('{}') within a NetworkPolicy.